### PR TITLE
[TACHYON-1519] Display permission information for ls command

### PR DESCRIPTION
**Pull request by shimingfei**
Display permission information for ls command
'd' represents it is a directory

d---------                    0.00B     01-07-2016 16:27:55:076                 /test/huge/tachyon-tiered-hdd-patched
d---------                    0.00B     01-07-2016 16:27:56:433                 /test/huge/tachyon-tiered-hdd-new
d---------                    0.00B     01-07-2016 16:27:57:929                 /test/huge/tachyon-tiered
d---------                    0.00B     01-07-2016 16:27:59:329                 /test/huge/mem-and-disk-ser
----------                     9.64MB    01-07-2016 16:28:01:113  In Memory      /test/huge/huge.tar.gz

### CODE REVIEW

File: shell/src/main/java/tachyon/shell/command/CommandUtils.java [hunk 1]
```diff
@@ -67,6 +67,42 @@ public static String convertMsToDate(long millis) {
   }
 
   /**
+   * Converts an int permission value to a formatted String.
+   *
+   * @param permission value of permission for the path
+   * @param isDir whether the path is a directory
+   * @return formatted permission String
+   */
+  public static String convertPermission(int permission, boolean isDir) {
```
Comment (yupeng9) [inline shell/src/main/java/tachyon/shell/command/CommandUtils.java]: Could you add some unit tests for this method?

**Comment by yupeng9** [comment]
Thanks for the improvement! Could you add some test coverage?

**Comment by shimingfei** [comment]
@yupeng9 
Sure, I will add some test cases.

**Comment by shimingfei** [comment]
@yupeng9 
I have added test cases. Thanks!

File: shell/src/main/java/tachyon/shell/command/CommandUtils.java (hunk 1 above)
Comment (yupeng9) [inline shell/src/main/java/tachyon/shell/command/CommandUtils.java]: Do you think `formatPermission` might be better?

**Comment by yupeng9** [comment]
@shimingfei  I left one comment about the naming. Thanks.

**Comment by shimingfei** [comment]
@yupeng9 I have renamed it. Thanks!

**Comment by yupeng9** [comment]
LGTM

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**